practice-one/src/styles/components/food-card.css

```diff
@@ -0,0 +1,62 @@
+.card-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 310px;
+  height: 516px;
```

Comment:
Height of the box with background gradient is not correct, please recheck the design

Reply:
i fixed it, thanks